### ISSUE TITLE
IntelliSense and auto completion not working in Ubuntu

### ISSUE THREAD

**Issue by jigarmistry** [Microsoft/vscode#30867]
It looks like the auto completion and intelliSense  is broken. It is working fine in vscode 1.13. Just installed that old version everything work fine.

VSCode Version: 1.14.1
OS Version: Ubuntu 16

Steps to Reproduce:

1. Open any project
1. Try to do a ctrl+mouse arrow to see the definition. It is not working.
1. Also auto completion is not working.
1. I am working node JS and react based projects

**Comment by mjbvz** [comment]
@Jakeii  Can you please:

1. Check the `TypeScript` section of the output panel. Are there any errors printed?
1. Open the VSCode developer tools by running the `Toggle Developer Tools` command. Look in the console. Are there any errors printed there?

**Comment by jigarmistry** [comment]
1. 

```
[Info  - 10:16:03 PM] Using tsserver from: /usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js
[Info  - 10:16:03 PM] Forking TSServer
PATH: /home/jigar/Documents/flutter/bin:/home/jigar/.yarn/bin:/home/jigar/bin:/home/jigar/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/lib/jvm/java-8-oracle/bin:/usr/lib/jvm/java-8-oracle/db/bin:/usr/lib/jvm/java-8-oracle/jre/bin:/opt/android-sdk/tools:/opt/android-sdk/platform-tools:/usr/lib/jvm/java-8-oracle/bin:/usr/lib/jvm/java-8-oracle/db/bin:/usr/lib/jvm/java-8-oracle/jre/bin
[Error  - 10:16:41 PM] 'quickinfo' request failed with error.
Error processing request. Project '/home/jigar/Documents/Projects/MyProject/jsconfig.json' does not contain document '/home/jigar/.cache/typescript/2.4/node_modules/parse5/node_modules/@types/node/index.d.ts'
Error: Project '/home/jigar/Documents/Projects/MyProject/jsconfig.json' does not contain document '/home/jigar/.cache/typescript/2.4/node_modules/parse5/node_modules/@types/node/index.d.ts'
    at Object.ThrowProjectDoesNotContainDocument (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:75718:23)
    at ConfiguredProject.Project.getScriptInfoForNormalizedPath (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:77238:42)
    at ConfiguredProject.Project.getScriptInfo (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:77243:29)
    at ModuleBuilder.AbstractBuilder.getOrCreateFileInfo (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:76551:51)
    at /usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:76679:84
    at Object.map (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:1483:29)
    at ModuleBuilder.getReferencedFileInfos (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:76679:31)
    at ModuleBuilder.updateFileReferences (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:76708:42)
    at ModuleBuilder.ensureProjectDependencyGraphUpToDate (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:76693:30)
    at ModuleBuilder.onProjectUpdateGraph (/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:76684:22)
```
2.

`/usr/share/code/resources/app/out/vs/workbench/electron-browser/workbench.main.js:29 [Extension Host] (node:15595) DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
h.logExtensionHostMessage @ /usr/share/code/resources/app/out/vs/workbench/electron-browser/workbench.main.js:29`

**Comment by mjbvz** [comment]
Thanks. Only the first error is relevant here. Looks like this is a duplicate of https://github.com/Microsoft/TypeScript/issues/17131

As a workaround, try disabling type acquisition as [suggested here](https://github.com/Microsoft/vscode/issues/30425#issuecomment-315660909). You can also downgrade the TypeScript version. [Here are the instructions on that](https://code.visualstudio.com/Docs/languages/typescript#_using-newer-typescript-versions). We previously shipped with TS 2.3.4 so you'll want to try installing that version locally and using it in your workspace
